package.json

```diff
@@ -36,19 +35,18 @@
     "test:e2e:playground": "cypress run --spec cypress/integration/playground/**/*.js",
     "test:e2e:template-starter": "cypress run --spec cypress/integration/template-starter/**/*.js",
     "vrt:components": "percy exec -- yarn test:visual",
-    "playground:build": "yarn --cwd playground build",
-    "playground:start": "yarn --cwd playground start",
-    "playground:start:prod:local": "yarn --cwd playground start:prod:local",
-    "visual-testing-app:start": "yarn --cwd visual-testing-app start",
-    "visual-testing-app:build": "yarn --cwd visual-testing-app build",
-    "visual-testing-app:serve": "yarn --cwd visual-testing-app serve",
-    "template-starter:build": "yarn --cwd application-templates/starter build",
-    "template-starter:start": "yarn --cwd application-templates/starter start",
-    "template-starter:start:prod:local": "yarn --cwd application-templates/starter start:prod:local",
-    "prebuild": "lerna run prebuild --no-private",
-    "build": "preconstruct build && NODE_ENV=production lerna run build --no-private",
-    "build:watch": "preconstruct watch & NODE_ENV=development lerna run --no-private --parallel build:bundles:watch",
-    "build:website": "yarn build && yarn --cwd website build && yarn --cwd website-components-playground build",
+    "playground:build": "yarn manypkg run playground build",
+    "playground:start": "yarn manypkg run playground start",
+    "playground:start:prod:local": "yarn manypkg run playground start:prod:local",
+    "visual-testing-app:start": "yarn manypkg run visual-testing-app start",
+    "visual-testing-app:build": "yarn manypkg run visual-testing-app build",
+    "visual-testing-app:serve": "yarn manypkg run visual-testing-app serve",
+    "template-starter:build": "yarn manypkg run application-template-starter build",
+    "template-starter:start": "yarn manypkg run application-template-starter start",
+    "template-starter:start:prod:local": "yarn manypkg run application-template-starter start:prod:local",
+    "build": "preconstruct build && NODE_ENV=production yarn manypkg run application-template-starter build && NODE_ENV=production yarn manypkg run mc-html-template build && NODE_ENV=production yarn manypkg run mc-scripts build && NODE_ENV=production yarn manypkg run playground build && NODE_ENV=production yarn manypkg run visual-testing-app build && NODE_ENV=production yarn manypkg run website build && NODE_ENV=production yarn manypkg run website-components-playground build",
```

Comment:
This is the explicit version of this confusing bit here

https://github.com/commercetools/merchant-center-application-kit/blob/main/package.json#L48-L49

We run prebuild in all packages. To run build with manypkg which runs prebuild again in some packages. Then we run build in all packages using lerna. Lerna only runs them in the packages which have it explicitly defined. Which runs prebuild again.

Reply:
The applications shouldn't be built here. Only the "nodejs" packages.

The `build` command is only meant to build library packages (see the "no private" option from the usage with lerna).